taskcluster/win-opt-base.tyml

```
@@ -0,0 +1,89 @@
+      TENSORFLOW_BUILD_ARTIFACT: ${build.tensorflow}
+      DEEPSPEECH_TEST_MODEL: https://queue.taskcluster.net/v1/task/${training}/artifacts/public/output_graph.pb
+
+  command:
+    - >-
+      "C:\Program Files\7-zip\7z.exe" x -txz -so msys2-base-x86_64.tar.xz |
+      "C:\Program Files\7-zip\7z.exe" x -o%USERPROFILE% -ttar -aoa -si
+    - .\msys64\usr\bin\bash.exe --login -cx "exit"
+    - .\msys64\usr\bin\bash.exe --login -cx "pacman --noconfirm -Syu"
+    - echo .\msys64\usr\bin\bash.exe --login -cxe "
+      export LC_ALL=C &&
+      export PATH=\"/c/builds/tc-workdir/msys64/usr/bin:/c/Python36:/c/Program Files/Git/bin:/c/Program Files/7-Zip/:$PATH\" &&
+      export TASKCLUSTER_ARTIFACTS=\"$USERPROFILE/public\" &&
+      export TASKCLUSTER_TASK_DIR=\"/c/builds/tc-workdir/\" &&
+      (mkdir $TASKCLUSTER_TASK_DIR || rm -fr $TASKCLUSTER_TASK_DIR/*) && cd $TASKCLUSTER_TASK_DIR &&
+      env &&
+      ln -s $USERPROFILE/msys64 $TASKCLUSTER_TASK_DIR/msys64 &&
+      (wget -O - $TENSORFLOW_BUILD_ARTIFACT | 7z x -txz -si -so | 7z x -o$TASKCLUSTER_TASK_DIR -aoa -ttar -si ) &&
+      git clone --quiet ${event.head.repo.url} $TASKCLUSTER_TASK_DIR/DeepSpeech/ds/ &&
+      cd $TASKCLUSTER_TASK_DIR/DeepSpeech/ds && git checkout --quiet ${event.head.sha} &&
+      ln -s $TASKCLUSTER_TASK_DIR/DeepSpeech/ds/native_client/ $TASKCLUSTER_TASK_DIR/DeepSpeech/tf/native_client &&
+      cd $TASKCLUSTER_TASK_DIR &&
+      pacman --noconfirm -R bsdtar &&
+      pacman --noconfirm -S tar make &&
+      $TASKCLUSTER_TASK_DIR/DeepSpeech/ds/${build.scripts.build} &&
+      $TASKCLUSTER_TASK_DIR/DeepSpeech/ds/${build.scripts.package} ;
+      echo \"export TASKCLUSTER_TASK_EXIT_CODE=$?\" > $USERPROFILE/tc-exit.sh &&
+      cd $TASKCLUSTER_TASK_DIR/../ && rm -fr tc-workdir/ && exit $TASKCLUSTER_TASK_EXIT_CODE" | cmd
+      /k ""C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat"" x64
+
+    - .\msys64\usr\bin\bash.exe --login -cxe "source $USERPROFILE/tc-exit.sh &&
```

Because of the trick above, we were not able to properly `exit $TASKCLUSTER_EXIT_CODE`, this would get mangled into `cmd.exe` so the easiest is to output it into a shell in the user's profile (which is different for each scheduled task, so no overlapping), and return it this way. It helps avoiding silent failures.